model/tab/gear/gear-main.html

```diff
@@ -0,0 +1,19 @@
+<div class="main border">
+    <div class="weight">
+        <label>{{localize "GEAR.WEIGHT"}}</label>
+        <select name="data.weight">
+            {{#select data.weight}}
+                <option value="none">{{localize "WEIGHT.NONE"}}</option>
+                <option value="tiny">{{localize "WEIGHT.TINY"}}</option>
+                <option value="light">{{localize "WEIGHT.LIGHT"}}</option>
+                <option value="regular">{{localize "WEIGHT.REGULAR"}}</option>
+                <option value="heavy">{{localize "WEIGHT.HEAVY"}}</option>
+            {{/select}}
+        </select>
+    </div>
+    <div class="features"><label>{{localize "GEAR.EFFECT"}}</label><textarea
+            name="data.effect">{{data.effect}}</textarea></div>
```

Comment:
This is problematic. Now you have two input fields for `effect` on the gear sheet (since it's also on the artifact tab). The result is not good. :)

Reply:
Yes. This is something I was going back and forth with various solutions to.

In the end. Sourcebook uses the term "Effect" for gear and "Effects" for artifacts. 

There is no good solution here that I can see. Removing/Replacing "effect" from normal gear would require migration, while adding a new "effects" property to artifacts would also require migration.

The reason I decided to go down this road is that a user is unlikely to use the effect field for separate descriptions. E.g. an Artifact with a normal gear Effect as well. UX-wise its not super appealing, however.

Any suggestions?

Reply:
My suggestion would be to add a conditional to the artifact tab to not render the effect field there for items of type `gear`.

Right now the two effect fields get concatenated with a comma in between every time you open the sheet. 😛